src/features/data/chains.ts

```diff
@@ -180,13 +202,14 @@ export const CHAINS: Chain[] = [
         explorerUrl: "https://arbiscan.io/address/%s",
         networkType: "mainnet",
         rddUrl: "https://reference-data-directory.vercel.app/feeds-ethereum-mainnet-arbitrum-1.json",
-        tags: ["nftFloorPrice"],
+        queryString: "arbitrum-mainnet",
```

Comment:
```suggestion
        queryString: "arbitrum-mainnet",
        tags: ["nftFloorPrice"],
```

This needs to remain enabled.

Reply:
Perhaps it was my accidental mistake 😕 Thank you for fixing it!